randovania/games/prime2/json_data/Agon Wastes.txt

```diff
@@ -1889,8 +1885,15 @@ Extra - in_dark_aether: True
       Dark Agon Key 1 and Dark Agon Key 2 and Dark Agon Key 3 and Morph Ball Bomb and Morph Ball and Space Jump Boots and Bomb Space Jump (Expert) and Single Room Out of Bounds (Expert) and Slope Jump (Expert) and Standable Terrain (Expert) and Dark World Damage ≥ 175
   > Event - Amorbis
       All of the following:
-          Morph Ball Bomb and Morph Ball
-          Space Jump Boots or Bomb Jump (Beginner)
+          Morph Ball Bomb and Morph Ball and Dark World Damage ≥ 50 and Activate Safe Zone
+          Any of the following:
+              All of the following:
+                  Charge Beam and Dark World Damage ≥ 100
+                  Any of the following:
+                      Shoot Supers
+                      Light Beam and Light Ammo ≥ 60
```

Comment:
It doesn't really matter but you could move the charge beam requirement to the light beam path instead

Reply:
Agreed. Will update.